draft-ietf-shmoo-remote-fee.md

```diff
@@ -80,8 +80,10 @@ order to reach consensus on this or a similar principle that the IETF Administra
 can use to guide future decision about registration fees for full online meetings.
 
 The principle this document states is simple: there must always be an option for free
-remote participation in any IETF meeting and related events that are part of the IETF's
-open process {{!RFC3935}}, whether or not that meeting has a physical presence.
+remote participation in any IETF meeting,
+whether or not that meeting has a physical presence.
```

Comment:
As discussed on the list, this goes beyond the SHMOO charter. Given the other PRs, is this one overtaken by events?

Reply:
Does this change work?
```
there must always be an option for free
remote participation in any IETF meeting, as defined in {{!RFC8719}}.
```

Reply:
I just want to note again that this part about "whether or not that meeting has a physical presence" was added based on group discussion. Just removing it would revert that discussion. Maybe we need to bring is back to the list with a separate thread to ensure we have consensus to any change here?

Reply:
Yes.  You want to post a note or me?

Reply:
Go ahead if you want.